common.json

```diff
@@ -1,5 +1,11 @@
 {
-    "AzurePowerShell": [
+  "AzurePowerShell": [
```

Comment:
The indentation for your changes is completely different from the rest of the file.